### DISCUSSION TITLE
Documentation missing for Server-addon?

### DISCUSSION THREAD

**Discussion by JoackimPennerup** [storybookjs/storybook#15120] [Documentation]
I'd like to try out the addon for server-rendered components. However, I can not find any documentation that guides me on how to initialize a storybook for "server". Running `npx -p @storybook/cli sb init -t server` fails as it cannot find the type specified.

**Comment by shilman** [comment]
Use: `npx sb@next init -t server`

@jonspalmer what's the state of the documentation?

**Comment by e0ipso** [comment]
I am too interested in learning more about this. I am pushing an effort in the Drupal community to use this integration, but I am facing questions from people I don't know how to answer.

Knowing more, even if it's just pointers to the code where the code is for the server-generated components would be super helpful!

**Comment by joeslazaro** [comment]
I found the documentation for this here: https://github.com/storybookjs/storybook/blob/next/code/frameworks/server-webpack5/README.md

The related code directories that I have found in the repository are:
- code/frameworks/server-webpack5
- code/renderers/server

The strange thing is that the documentation is missing when you view https://www.npmjs.com/package/@storybook/server, but if you go to an older version then it is there: https://www.npmjs.com/package/@storybook/server/v/6.0.0
